knowledge_storm/storm_wiki/modules/article_polish.py

```diff
@@ -85,14 +85,14 @@ def __init__(
         self.polish_page = dspy.Predict(PolishPage)
 
     def forward(self, topic: str, draft_page: str, polish_whole_page: bool = True):
-        with dspy.settings.context(lm=self.write_lead_engine):
+        with dspy.settings.context(lm=self.write_lead_engine, show_guidelines=False):
```

Comment:
Add a small comment explaining this since this is changing our existing package, others may be confused why this needs to be changed.